templates/partials/question-definition.html

```diff
@@ -17,7 +22,8 @@
           "data-ga": "click",
           "data-ga-category": "Question definition",
           "data-ga-action": "Close panel",
-          "data-ga-label": definition.title
+          "data-ga-label": definition.title,
+          "data-qa": "question-definition-button-" ~ loop.index
```

Comment:
The prefixes for these attributes should be consistent, so this would be:
```suggestion
          "data-qa": "question-definition-"  ~ loop.index ~ "-button"
```